lib/brainstem/presenter_collection.rb

```diff
@@ -275,7 +278,16 @@ def run_search(scope, includes, sort_name, direction, options)
         search_options[:page] = calculate_page(options)
       end
 
-      search_options.reverse_merge!(extract_filters(options))
+      extracted_filters = extract_filters(options)
+      extracted_filters.each do |key, val|
+        if val[:arg].nil?
+          extracted_filters.delete(key)
```

Comment:
I'm not entirely sure I understand what they represent, but it does seem to be expected that filters with nil args aren't included in the extracted filters hash here. (`extract_filters` used to do it like this, before these changes: `filters_hash[filter_name] = args unless args.nil?`)

I didn't know that`.each.with_object` trick, that's pretty nifty.